test/Microsoft.AspNet.Mvc.ViewFeatures.Test/ViewFeatures/DefaultDisplayTemplatesTest.cs

```diff
@@ -138,7 +138,7 @@ public void ObjectTemplate_IgnoresPropertiesWith_ScaffoldColumnFalse()
             var result = DefaultDisplayTemplates.ObjectTemplate(htmlHelper);
 
             // Assert
-            Assert.Equal(expected, HtmlContentUtilities.HtmlContentToString(result));
+            Assert.Equal(expected, HtmlContentUtilities.HtmlContentToString(result), ignoreLineEndingDifferences: true);
```

Comment:
Because the rest of them are constructed using Environment.NewLine and this one is constructed using @"". I'll rewrite it to more closely match the other tests.